syntax = "proto3";

package task;

service SendTask
{
	// A simple RPC.
	//
	// Obtains the MessageResponse at a given position.
   rpc GetServerResponse(Task) returns (TaskResponse) {}
  
  }

service ManageWorkers
{
	rpc GetWorkersResponse(Workers) returns (WorkersResponse) {}
}

message Task {
	string task = 1;
	string file = 2;
}

message TaskResponse {
	string result = 1;
}

message Workers {
	string manage = 1;
}

message WorkersResponse {
	bool result = 1;
}

